### DISCUSSION TITLE
Private instagram accounts? Workarounds?

### DISCUSSION THREAD

**Discussion by MWP** [karakeep-app/karakeep#503] [General]
Hi all,
New to hoarder, seems like a good idea :)

I've just come across my first use problem... hoarder can't grab pages/images from pvt instagram accounts.
Obviously this is due to the way hoarder grabs url/page data.

I wondering how you all work around this problem?

Thanks :)

**Comment by kamtschatka** [comment]
There is no workaround. There is however a feature request to allow scraping the webpage from the browser: https://github.com/hoarder-app/hoarder/issues/172

**Reply by replicater233** [reply]
When can we expect the request to be accepted?
